src/dx/formatters/summarizing.py

```diff
@@ -0,0 +1,53 @@
+from typing import Callable, Optional
+
+import pandas as pd
+
+
+class DataFrameSummarizer:
+    _instance: "DataFrameSummarizer" = None
+    summarizing_func: Optional[Callable] = None
+
+    def __init__(self, summarizing_func: Optional[Callable] = None):
```

Comment:
If they set it here, shouldn't we _not_ load repr llm's version?

Reply:
Good point - updating